### ISSUE TITLE
Mouse movement using significant CPU time

### ISSUE THREAD

**Issue by nstlaurent** [opentk/opentk#1652]
### Description
On slower systems we have been experiencing severe FPS drops whenever the mouse is moved. Profiling has showed GLFW.GetCursorPos using significant CPU time. Requires moving the mouse to show up, leaving mouse in the same spot has no issue. This problem has even showed up on an i7-8750h.

On my development laptop running a Ryzen 5900HS, GLFW.GetCursorPos has been shown to be using as much as 1.5%.

JetBrains DotTrace with no mouse movement:
![nomousemove](https://github.com/opentk/opentk/assets/42744211/251a3b68-c3d0-42ba-8f37-fcb2a30f48c7)

JetBrains DotTrace with mouse movement:
![movemouse](https://github.com/opentk/opentk/assets/42744211/a158034c-f6cc-45e7-a5e3-463fe1ea2823)

We have also tried using GLFW directly to use raw input in Windows, but it didn't make a difference:
```
if (GLFW.RawMouseMotionSupported())
    GLFW.SetInputMode(window.WindowPtr, RawMouseMotionAttribute.RawMouseMotion, true);
```

I would find it acceptable if there was option to disable mouse processing in OpenTK. Currently, it appears to be processed even if you don't register for the mouse events. We have our own implementation using Windows Raw Input as most of our users are on Windows. GLFW doesn't seem to expose the fact that Windows Raw Input is event driven anyway, so we would just prefer to roll the code ourselves and disable mouse processing in OpenTK.

Simple example program:
```
using OpenTK.Graphics.OpenGL;
using OpenTK.Mathematics;
using OpenTK.Windowing.Common;
using OpenTK.Windowing.Desktop;

GameWindowSettings gwSettings = new GameWindowSettings()
{
};

NativeWindowSettings nwSettings = new NativeWindowSettings()
{
    API = ContextAPI.OpenGL,
    APIVersion = new Version(3, 3),
    AutoLoadBindings = true,
    Flags = ContextFlags.Debug | ContextFlags.ForwardCompatible,
    IsEventDriven = false,
    Profile = ContextProfile.Core,
    Size = (800, 600),
    StartFocused = true,
    StartVisible = true,
    Title = "Local OpenTK Test",
    WindowBorder = WindowBorder.Resizable,
    WindowState = WindowState.Normal,
};

using (TestWindow window = new TestWindow(gwSettings, nwSettings))
{
    window.Run();
}

class TestWindow : GameWindow
{
    public TestWindow(GameWindowSettings gwSettings, NativeWindowSettings nwSettings) : base(gwSettings, nwSettings)
    {
    }

    protected override void OnLoad()
    {
        base.OnLoad();
    }

    protected override void OnRenderFrame(FrameEventArgs args)
    {
        base.OnRenderFrame(args);

        GL.ClearColor(Color4.Black);
        GL.Clear(ClearBufferMask.ColorBufferBit | ClearBufferMask.DepthBufferBit);

        SwapBuffers();
    }

    protected override void OnUpdateFrame(FrameEventArgs args)
    {
        base.OnUpdateFrame(args);
    }
}
```

### Expected behavior
Moving the mouse should not use significant CPU.

### Actual behavior
Moving the mouse uses significant CPU.

### Related information 

Windows 10/Windows 11
OpenTK 4.8.1
.NET 7

**Comment by NogginBops** [comment]
It's interesting that the mouse position polling takes CPU time as that operation should only happen once per frame. And the glfw implementation is really quite simple...
![image](https://github.com/opentk/opentk/assets/3778357/a299e8b7-9e5d-49fc-86d4-55ea8863f9d6)

Would like to avoid having to have a setting for removing mouse input updates. Would be much better to try find the root cause.
I'll try to repro the issue myself and see what I can find out.

**Comment by NogginBops** [comment]
There has been no work nor news around this issue and we want to get `4.8.2` out with some important regression fixes so I'm moving this to `4.8.3`.

**Comment by NogginBops** [comment]
If you want to stop `NativeWindow` or `GameWindow` from processing input I would recommend using `NativeWindow.ProcessWindowEvents` manually instead of using the `Run()` method.
Input processing is done in `NativeWindow.NewInputFrame()` which is called by `NativeWindow.ProcessEvents` and `GameWindow.Run()`.

Going to close this issue with this as I think this should fix the issue you are experiencing. Feel free to reopen the issue if that is not the case.
